### ISSUE TITLE
The docs don't mention salesforce integration

### ISSUE THREAD

**Issue by rockymeza** [fusionbox/django-widgy#301]
That seems like something of a selling point.

**Comment by plee** [comment]
We need end user docs of how to use the widget also.  It was difficult for me to know that we need to map the fields. :octopus:

**Comment by rockymeza** [comment]
good point, would you like to write those? I can help you getting set up
with the reStructuredText.

*Rocky Meza*, Programmer
fusionbox*▝*
o: 303.952.7490

www.fusionbox.com
<http://www.fusionbox.com/?utm_source=fb%2Bemail&utm_medium=email&utm_campaign=email%2Btracking>

This information is confidential and intended solely for the use
of the individual or entity to whom it is addressed. If you have
received this email in error please notify the sender.
